src/vorta/utils.py

```diff
@@ -31,6 +33,48 @@
 logger.info('Using %s Keyring implementation.', keyring.__class__.__name__)
 borg_compat = BorgCompatibility()
 
+class FilePathInfoAsync(QThread):
+    signal = pyqtSignal(str,int,int)
+    def __init__(self,path):
+        self.path = path
+        QThread.__init__(self)
+        self.exiting = False
+
+    def run(self):
+        self.files_count = 0
+        self.size, self.files_count = get_path_datasize(self.path)
+        self.signal.emit(self.path,self.size,self.files_count)        
```

Comment:
```suggestion
        self.signal.emit(self.path,str(self.size),str(self.files_count))        
```

Reply:
Don't really get this: is that related to "The size is incorrect, du -sb /home/user produces 270764M when run from command line, shows 447.4M in Vorta. This is caused by the size being greater than the C++ int limit, convert it to pretty_bytes before returning it."?

Reply:
Yeah, converting it to string returning it, and then converting back to int prevents overflow.

This is because Python 3 int does not have a maximum, while C++ signed int does have a maximum (2^31-1). 
Qt 5 is C++ based, PyQt5 is just a binding that calls the C++ code.